### ISSUE TITLE
Hot reload not working on React Email 2.1.0

### ISSUE THREAD

**Issue by danieljaguiar** [resend/react-email#1359]
### Describe the Bug

Updating emails like "notion-magic-link" doesn't reflect on web browser until refreshing the page

### Which package is affected (leave empty if unsure)

react-email

### Link to the code that reproduces this issue

https://github.com/danieljaguiar/test-hot-reload-react-email

### To Reproduce

1. pnpm dlx create-next-app
2. cd "project name"
3. pnpm create email
4. cd .\react-email-starter\
5. pnpm install
6. pnpm run dev
7. update react-email-starter\emails\notion-magic-link.tsx

### Expected Behavior

See the changes on "react-email-starter\emails\notion-magic-link.tsx" in the browser

**Comment by gabrielmfern** [comment]
What OS are you on?

**Comment by danieljaguiar** [comment]
> What OS are you on?

Windows 11

**Comment by gabrielmfern** [comment]
Hey @danieljaguiar, tried reproducing locally here and seems like the latest canary (i.e. `react-email@2.1.1-canary.1`) has this fixed as well, probably due to #1356.
Can you check this on your end as well? You can upgrade the packages by just re-installing them like:

```sh
pnpm add @react-email/components@canary react-email@canary -E
```

**Comment by danieljaguiar** [comment]
@gabrielmfern it is fixed in `2.1.1-canary.1`

Thank you!

**Comment by gabrielmfern** [comment]
Lovely! Windows is a big priority for us, if you do find anything else that might feel like an inconvenience to you here, please don't hesitate in opening an issue.

If anyone comes across this and wants to just start up a new project from canary, the latest canary for `create-email` has its template using all the latest canaries as well. Just do `npx create-email@canary` and you're set to go.
